scripts/archive_leg_to_csv.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+import csv
+import django
+from django import conf
+import click
+from utils import (
+    get_jurisdiction_id,
+)
+
+
+def init_django():  # pragma: no cover
+    conf.settings.configure(
+        conf.global_settings,
+        SECRET_KEY="not-important",
+        DEBUG=False,
+        INSTALLED_APPS=(
+            "django.contrib.contenttypes",
+            "opencivicdata.core.apps.BaseConfig",
+            "opencivicdata.legislative.apps.BaseConfig",
+        ),
+        DATABASES={
```

Comment:
let's pull this from the environment like we do in to_database.py

Reply:
actually, let's refactor here and move init_django into utils maybe so we can just share it between the two

Reply:
Moved into utils.